src/content/blog.njk

```diff
@@ -16,7 +16,7 @@ pagination:
           <time class="date" datetime="{{ post.date | readableDate }}">{{ post.date | readableDate }}</time>
         </p>
         <h2 class="post-title">
-          <a class="" href="/blog/{{ post.date | luxon('y/LL/dd') }}/{{ post.slug }}" class="postlist-link">{% if post.title.rendered %}{{ post.title.rendered | safe }}{% else %}{{ post.slug }}{% endif %}</a>
+          <a class="" href="{{ post.postUrl }}" class="postlist-link">{% if post.title.rendered %}{{ post.title.rendered | safe }}{% else %}{{ post.slug }}{% endif %}</a>
```

Comment:
`postUrl` was a terrible name, I meant to change it. But good idea to use it instead of the duplication.

Reply:
I think WP uses `permalink`, would that work for you?